CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased] - 2024-??-??
+## [0.11.0] - 2024-09-18
```

Comment:
Do we have a strategy when it comes to minor and major versions? 0.11.0 sounds ok to me because we also add functionality here besides just bug fixing, but good to be on the same page...